src/main/java/org/jenkinsci/plugins/credentialsbinding/impl/ZipFileBinding.java

```diff
@@ -73,6 +78,11 @@ public FormValidation doCheckCredentialsId(@AncestorInPath Item owner, @QueryPar
                     } catch (IOException x) {
                         return FormValidation.warning("Could not verify file format");
                     }
+                    finally {
+                        if (is != null) {
+                            IOUtils.closeQuietly(is);
```

Comment:
:ant: better style is to use

```java
try {
    InputStream is = c.getContent();
    try {
        // use it…
    } finally {
        is.close();
    }
} catch (IOException x) {
    return FormValidation.warning("Could not verify file format");
}
```

since this avoids use of a non-blank-final variable as well as the undesirable `closeQuietly`, and allows for easy mechanical translation (e.g., IDE hint) to the right idiom when you upgrade to `-source 7` or newer:

```java
try {
    try (InputStream is = c.getContent()) {
        // use it…
    }
} catch (IOException x) {
    return FormValidation.warning("Could not verify file format");
}
```